index.js

```diff
@@ -1,4 +1,32 @@
 import { createServer } from "http";
+import { Octokit } from "@octokit/rest";
+
+const ACCESS_TOKEN = process.env.GITHUB_PERSONAL_ACCESS_TOKEN;
+
+const octokit = new Octokit({
+  auth: ACCESS_TOKEN,
+});
+
+const httpServer = createServer(async (_, response) => {
+  const repos = await getRepos({ org: "dxw" });
+
+  return response.end(JSON.stringify(repos));
+});
+
+const getRepos = async ({ org }) => {
+  return octokit
+    .request("GET /orgs/dxw/repos", {
+      org,
+    })
```

Comment:
Do we want to hardcode the org or parameterise it? Because it looks like we're doing both here - I'm not sure what Octokit does in this situation.

Reply:
good spot, I think we should parameterize it. It's almost no extra hassle and will make things easier if we want to support different orgs in future.